### DISCUSSION TITLE
Real-time API support

### DISCUSSION THREAD

**Discussion by Bodekaer** [RageAgainstThePixel/com.openai.unity#302] [Q&A] [answered]
Hey @StephenHodgson,
Here we go again. A new day and a new API to support :D haha.
How are you feeling about/planning for the new Real-time API? I'm guessing you're already all over it? 
I'm very excited to start using it/helping where possible. It's a game-changer for our applications :) 

https://platform.openai.com/docs/api-reference/realtime

Btw. I missed the colab invitation you sent, so it expired. Please just re-send.

**Comment by StephenHodgson** [comment, marked as answer]
yup I have already been working on it. Tracking issue: 

-  https://github.com/RageAgainstThePixel/OpenAI-DotNet/issues/371

**Reply by StephenHodgson** [reply]
Did some work last weekend, but got stuck with WebSocket support in webgl build target.
